main/2021/WG-03-10.md

```diff
@@ -0,0 +1,46 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the March 10th video call of WebAssembly's Working Group
+
+- **Where**: zoom.us
+- **When**: March 10, 2021 at 4pm-5pm UTC *( March 10th, 8am-9am PST )*
+- **Location**: *on calendar invite to registered attendees*
+- **Contact**:
+    - Name: Derek Schuff, Luke Wagner
+
+### Registration
+
+If you are a Working Group member no registration is required.
+
+If you are a Community Group member who would like to observe, please register here: https://goo.gl/forms/HD2kLCM0iSKk7AVl1
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
```

Comment:
This actually isn't true, since Zoom has a web client.

Reply:
Oh hah, right; I just copy-pasted.  Should it just say "See the calendar invite for link"?

Reply:
Yeah that makes sense.